lib/operations/connect.js

```diff
@@ -334,30 +333,7 @@ function createTopology(mongoClient, options, callback) {
       return;
     }
 
-    const mongoCryptOptions = Object.assign(
-      {
-        bson:
-          options.bson ||
-          new BSON([
-            BSON.Binary,
-            BSON.Code,
-            BSON.DBRef,
-            BSON.Decimal128,
-            BSON.Double,
-            BSON.Int32,
-            BSON.Long,
-            BSON.Map,
-            BSON.MaxKey,
-            BSON.MinKey,
-            BSON.ObjectId,
-            BSON.BSONRegExp,
-            BSON.Symbol,
-            BSON.Timestamp
-          ])
-      },
-      options.autoEncryption
-    );
-
+    const mongoCryptOptions = Object.assign({}, options.autoEncryption);
```

Comment:
the CSFLE module will need a reference to `bson` passed in still, I think you can just use `{ bson: BSON }` here.